renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:best-practices"
```

Comment:
I like this 👍 

References: 
- https://docs.renovatebot.com/upgrade-best-practices/
- https://github.com/open-telemetry/sig-security/issues/69

Reply:
Yeah, I wasn't sure if more config was needed to enable it. I was planning to see what happens when we run this and if it doesn't update as expected make modifications to the config.